src/singlyLinkedList/index.ts

```diff
@@ -0,0 +1,217 @@
+import { SinglyLinkedNode as Node } from './node';
+
+export class SinglyLinkedList<T> {
```

Comment:
I've been going through [this Udemy course](https://www.udemy.com/course/js-algorithms-and-data-structures-masterclass/) and have been adopting / modifying the methods the instructor teaches. One thing tho, I think I may change the method names to be more descriptive. E.g. `insertAtEnd` instead of `push` OR at least make aliases for those methods. Even in working on the class I had to keep asking which is which hahah. Any thoughts on method names?